tests/integration/targets/aci_vmm_credential/tasks/vmware.yml

```diff
@@ -19,232 +19,240 @@
     state: absent
   register: nm_remove_domain
 
-# ADD VMM domain for testing
-- name: Add VMM domain (normal mode)
-  cisco.aci.aci_domain: &domain_present
-    host: '{{ aci_hostname }}'
-    username: '{{ aci_username }}'
-    password: '{{ aci_password }}'
-    validate_certs: '{{ aci_validate_certs | default(false) }}'
-    use_ssl: '{{ aci_use_ssl | default(true) }}'
-    use_proxy: '{{ aci_use_proxy | default(true) }}'
-    output_level: '{{ aci_output_level | default("info") }}'
-    domain: vmm_dom
-    domain_type: vmm
-    vm_provider: vmware
-    state: present
-  register: nm_add_domain
+- name: Verify Cloud and Non-Cloud Sites in use.
+  include_tasks: ../../../../../../integration/targets/aci_cloud_provider/tasks/main.yml
 
-- name: Verify add_domain
-  assert:
-    that:
-    - nm_add_domain is changed
+- name: Execute tasks only for non-cloud sites
+  when:
+  - version.current.0.topSystem.attributes.version is version('4.1', '>=')
+  - query_cloud.current == []
+  block:
+  # ADD VMM domain for testing
+  - name: Add VMM domain (normal mode)
```

Comment:
Can we add the aci_info section here and use it in this test if we modify it anyway?